titan-options.php

```diff
@@ -489,44 +495,1683 @@ function regala_create_options() {
 	/**
 	 *   Sections
 	 */
+
+    // Section 1
+    $section1 = $titan->createThemeCustomizerSection( array(
+        'name' => __( 'Section 1 - Layout', 'havana' ),
+        'panel' =>  __('Section 1', 'havana'),
+    ) );
     
-    $havanaSections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Sections', 'regala' ),
-        'panel' => __( 'Theme Options & Colors', 'regala' ),
+    $section1->createOption( array(
+        'panel' => __( 'Section 1', 'havana' ),
+        'id' => 'section1_layout',
+	    'type' => 'radio-image',
+	    'options' => array(
+             'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
+             'layout2' => get_template_directory_uri() . '/images/header-image-default1.jpg',
+             'layout3' => get_template_directory_uri() . '/images/header-image-default2-thumb.jpg',
+             'layout4' => get_template_directory_uri() . '/images/header-image-default2.jpg',
+             'layout5' => get_template_directory_uri() . '/images/logo.png',
+             'layout6' => get_template_directory_uri() . '/images/site-icon.png',
+         ),
     ) );
     
-    $sections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Section 1', 'regala' ),
-        'panel' => __( 'Sections', 'regala' ),
+    $section1Box1 = $titan->createThemeCustomizerSection( array(
+        'name' => __( 'Section 1 - Box 1 Content', 'havana' ),
+        'panel' =>  __('Section 1', 'havana'),
     ) );
     
-    $sections->createOption( array(
-        'name' => __( 'Layout', 'regala' ),
-        'id' => 'section_1',
-        'type' => 'radio-image',
-        'options' => array(
-            'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
-            'layout2' => get_template_directory_uri() . '/images/header-image-default1.jpg',
-            'layout3' => get_template_directory_uri() . '/images/header-image-default2-thumb.jpg',
-            'layout4' => get_template_directory_uri() . '/images/header-image-default2.jpg',
-            'layout5' => get_template_directory_uri() . '/images/logo.png',
-            'layout6' => get_template_directory_uri() . '/images/site-icon.png',
-         ),
-    ) );    
-  
+    $section1Box1->createOption( array(
```

Comment:
Since all of these are just repeated options. You can loop the createOption process per section to make the code shorter